customization/dark-theme.md

```diff
@@ -120,7 +120,7 @@ The advantage of creating a dark themed Template VM is, that each AppVM which is
 
 1. Start VM
 
-    **Note:** In the case of App VM start the Template on which the AppVM is based on.
+    **Note:** Remember that if you want to make the change persistent, the change needs to be made in the TemplateVM, not the AppVM.
```

Comment:
"If this is not a StandaloneVM, remember that the change needs to be made in the TemplateVM, not the AppVM, in order to persist across reboots" ?